src/runtime/particle-spec.ts

```diff
@@ -77,8 +77,7 @@ export class HandleConnectionSpec implements HandleConnectionSpecInterface {
   dependentConnections: HandleConnectionSpec[];
   pattern?: string;
   parentConnection: HandleConnectionSpec | null = null;
-  /** Maps from field path (including handle name, e.g. myHandle.someRef.someField) to list of claims. */
-  claims?: Map<string, Claim[]>;
+  claims?: ParticleClaim[];
```

Comment:
Mostly a clarification. Seems like this change is a step backwards? If you want to use the `ParticleClaim` class, shouldn't we move the `claims` to ParticleSpec class itself?

Reply:
It's already on the ParticleSpec class. It needs to be on the HandleConnectionSpec class too, I think this was used when building up the FlowGraph.

ParticleClaim is a poorly named class, but I don't have a better idea. I would just call it Claim, but that is already used as the type name for the specific kinds of claims you can make.

Reply:
Ah, gotcha. `Claim` would have been better and the current `Claim` could have been `ClaimType`. C'est la vie!